numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -3596,7 +3596,7 @@ def add_newdoc(place, name, doc):
 
     See Also
     --------
-    lib.scimath.sqrt
+    numpy.lib.scimath.sqrt
```

Comment:
I think the problem here isn't that it's missing the `numpy.`, but that there is no `lib.scimath.sqrt` article being generated to link to. I've added an `autosummary` to the module docstring following the pattern set in the polynomial modules to fix this problem in 31fbcf5a3

Reply:
Well seen!